src/core_modules/capture-core/metaDataMemoryStoreBuilders/programs/programsBuilder.js

```diff
@@ -63,16 +59,14 @@ async function getBuilderPrerequisites() {
     const cachedRelationshipTypesPromise = getRelationshipTypes(storageController, stores.RELATIONSHIP_TYPES);
     const cachedCategoriesPromise = getCategories(storageController, stores.CATEGORIES);
 
-    const values =
-        await Promise.all([
-            cachedProgramsPromise,
-            cachedProgramRulesVariables,
-            cachedProgramRules,
-            cachedProgramIndicatorsPromise,
-            cachedRelationshipTypesPromise,
-            cachedCategoriesPromise,
-        ]);
-    return values;
+    return Promise.all([
+        cachedProgramsPromise,
+        cachedProgramRulesVariables,
+        cachedProgramRules,
+        cachedProgramIndicatorsPromise,
+        cachedRelationshipTypesPromise,
+        cachedCategoriesPromise,
+    ]);
```

Comment:
Please correct me if I get this wrong here. I followed what eslint was suggesting. 

That basically was saying that in an async function awaiting for the promise to resolve and then return it is redundant since an async function always returns a promise anyways.

Reply:
Right